<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BookingDetails</class>
 <widget class="QWidget" name="BookingDetails">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1476</width>
    <height>973</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::CustomContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <property name="spacing">
    <number>30</number>
   </property>
   <property name="bottomMargin">
    <number>70</number>
   </property>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_21">
        <property name="font">
         <font>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: #3f4773</string>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;h1 align=&quot;center&quot; style=&quot; margin-top:18px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:xx-large; font-weight:700;&quot;&gt;BOOKING DETAIL&lt;/span&gt;&lt;/h1&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignBottom|Qt::AlignHCenter</set>
        </property>
        <property name="margin">
         <number>20</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="Frame">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
     <property name="midLineWidth">
      <number>0</number>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="spacing">
       <number>20</number>
      </property>
      <item>
       <widget class="QFrame" name="frame_5">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <property name="spacing">
          <number>50</number>
         </property>
         <property name="leftMargin">
          <number>30</number>
         </property>
         <property name="rightMargin">
          <number>30</number>
         </property>
         <item>
          <widget class="QFrame" name="frame_3">
           <property name="styleSheet">
            <string notr="true">border-width: 1;
border-style: none;
border-radius: 20;
background-color: white;</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout">
            <property name="spacing">
             <number>50</number>
            </property>
            <property name="leftMargin">
             <number>40</number>
            </property>
            <property name="topMargin">
             <number>50</number>
            </property>
            <property name="rightMargin">
             <number>40</number>
            </property>
            <property name="bottomMargin">
             <number>50</number>
            </property>
            <item>
             <widget class="QFrame" name="checkinFrame">
              <property name="styleSheet">
               <string notr="true">border-width: 1;
border-style: none;
border-bottom: 2px solid rgba(105,118,132,255);
border-left: 2px solid rgba(105,118,132,255);
border-radius: 10;
background-color: white;
</string>
              </property>
              <property name="frameShape">
               <enum>QFrame::Panel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Sunken</enum>
              </property>
              <property name="lineWidth">
               <number>1</number>
              </property>
              <property name="midLineWidth">
               <number>0</number>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout">
               <item>
                <widget class="QLabel" name="label_3">
                 <property name="maximumSize">
                  <size>
                   <width>56</width>
                   <height>56</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">border-style:none;</string>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                 <property name="pixmap">
                  <pixmap>images/user.png</pixmap>
                 </property>
                 <property name="scaledContents">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="clientNameLabel">
                 <property name="styleSheet">
                  <string notr="true">border-width: 1;
border-style: none;
font-size: 20px;
border-bottom: 2px solid rgba(105,118,132,255);
border-radius: 10;
background-color: white;
</string>
                 </property>
                 <property name="text">
                  <string>Nguyễn Việt Hưng</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QFrame" name="checkinFrame_8">
              <property name="styleSheet">
               <string notr="true">border-width: 1;
border-style: none;
border-bottom: 2px solid rgba(105,118,132,255);
border-left: 2px solid rgba(105,118,132,255);
border-radius: 10;
background-color: white;
</string>
              </property>
              <property name="frameShape">
               <enum>QFrame::Panel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Sunken</enum>
              </property>
              <property name="lineWidth">
               <number>1</number>
              </property>
              <property name="midLineWidth">
               <number>0</number>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <item>
                <widget class="QLabel" name="label_4">
                 <property name="maximumSize">
                  <size>
                   <width>56</width>
                   <height>56</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">border-style:none;</string>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                 <property name="pixmap">
                  <pixmap>images/telephone.png</pixmap>
                 </property>
                 <property name="scaledContents">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="phoneNumberLabel">
                 <property name="styleSheet">
                  <string notr="true">border-width: 1;
border-style: none;
font-size: 20px;
border-bottom: 2px solid rgba(105,118,132,255);
border-radius: 10;
background-color: white;
</string>
                 </property>
                 <property name="text">
                  <string>0987654312</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QFrame" name="checkinFrame_5">
              <property name="styleSheet">
               <string notr="true">border-width: 1;
border-style: none;
border-radius: 10;
background-color: white;
</string>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <item>
                <widget class="QLabel" name="label_7">
                 <property name="maximumSize">
                  <size>
                   <width>56</width>
                   <height>56</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">border-style:none;</string>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                 <property name="pixmap">
                  <pixmap>images/icons8-service-64.png</pixmap>
                 </property>
                 <property name="scaledContents">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QTableWidget" name="serviceTable">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">border: 1px solid black;
border-radius: 0;</string>
                 </property>
                 <property name="editTriggers">
                  <set>QAbstractItemView::NoEditTriggers</set>
                 </property>
                 <property name="tabKeyNavigation">
                  <bool>false</bool>
                 </property>
                 <property name="selectionMode">
                  <enum>QAbstractItemView::NoSelection</enum>
                 </property>
                 <column>
                  <property name="text">
                   <string>SERVICE</string>
                  </property>
                 </column>
                 <column>
                  <property name="text">
                   <string>ORDER TIME</string>
                  </property>
                 </column>
                 <column>
                  <property name="text">
                   <string>STATUS</string>
                  </property>
                 </column>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frame_4">
           <property name="styleSheet">
            <string notr="true">border-width: 1;
border-style: none;
border-radius: 20;
background-color: white;</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <property name="spacing">
             <number>30</number>
            </property>
            <property name="leftMargin">
             <number>40</number>
            </property>
            <property name="topMargin">
             <number>40</number>
            </property>
            <property name="rightMargin">
             <number>40</number>
            </property>
            <property name="bottomMargin">
             <number>40</number>
            </property>
            <item>
             <widget class="QFrame" name="checkinFrame_3">
              <property name="styleSheet">
               <string notr="true">border-width: 1;
border-style: none;
border-bottom: 2px solid rgba(105,118,132,255);
border-left: 2px solid rgba(105,118,132,255);
border-radius: 10;
background-color: white;
</string>
              </property>
              <property name="frameShape">
               <enum>QFrame::Panel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Sunken</enum>
              </property>
              <property name="lineWidth">
               <number>1</number>
              </property>
              <property name="midLineWidth">
               <number>0</number>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_4">
               <item>
                <widget class="QLabel" name="label_5">
                 <property name="maximumSize">
                  <size>
                   <width>56</width>
                   <height>56</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">border-style:none;</string>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                 <property name="pixmap">
                  <pixmap>images/checkin.png</pixmap>
                 </property>
                 <property name="scaledContents">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="clientCheckInLabel">
                 <property name="styleSheet">
                  <string notr="true">border-width: 1;
border-style: none;
font-size: 20px;
border-bottom: 2px solid rgba(105,118,132,255);
border-radius: 10;
background-color: white;
</string>
                 </property>
                 <property name="text">
                  <string>28/04/2022</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QFrame" name="checkinFrame_4">
              <property name="styleSheet">
               <string notr="true">border-width: 1;
border-style: none;
border-bottom: 2px solid rgba(105,118,132,255);
border-left: 2px solid rgba(105,118,132,255);
border-radius: 10;
background-color: white;
</string>
              </property>
              <property name="frameShape">
               <enum>QFrame::Panel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Sunken</enum>
              </property>
              <property name="lineWidth">
               <number>1</number>
              </property>
              <property name="midLineWidth">
               <number>0</number>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_6">
               <item>
                <widget class="QLabel" name="label_6">
                 <property name="maximumSize">
                  <size>
                   <width>56</width>
                   <height>56</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">border-style:none;</string>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                 <property name="pixmap">
                  <pixmap>images/checkout.png</pixmap>
                 </property>
                 <property name="scaledContents">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="clientCheckOutLabel">
                 <property name="styleSheet">
                  <string notr="true">border-width: 1;
border-style: none;
font-size: 20px;
border-bottom: 2px solid rgba(105,118,132,255);
border-radius: 10;
background-color: white;
</string>
                 </property>
                 <property name="text">
                  <string>01/05/2022</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QFrame" name="checkinFrame_9">
              <property name="styleSheet">
               <string notr="true">border-width: 1;
border-style: none;
border-bottom: 2px solid rgba(105,118,132,255);
border-left: 2px solid rgba(105,118,132,255);
border-radius: 10;
background-color: white;
</string>
              </property>
              <property name="frameShape">
               <enum>QFrame::Panel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Sunken</enum>
              </property>
              <property name="lineWidth">
               <number>1</number>
              </property>
              <property name="midLineWidth">
               <number>0</number>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_9">
               <item>
                <widget class="QLabel" name="label_10">
                 <property name="maximumSize">
                  <size>
                   <width>56</width>
                   <height>56</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">border-style:none;</string>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                 <property name="pixmap">
                  <pixmap>images/status-64.png</pixmap>
                 </property>
                 <property name="scaledContents">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="statusLabel">
                 <property name="styleSheet">
                  <string notr="true">border-width: 1;
border-style: none;
font-size: 20px;
border-bottom: 2px solid rgba(105,118,132,255);
border-radius: 10;
background-color: white;
</string>
                 </property>
                 <property name="text">
                  <string>Booked</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QFrame" name="checkinFrame_6">
              <property name="styleSheet">
               <string notr="true">border-width: 1;
border-style: none;
border-bottom: 2px solid rgba(105,118,132,255);
border-left: 2px solid rgba(105,118,132,255);
border-radius: 10;
background-color: white;
</string>
              </property>
              <property name="frameShape">
               <enum>QFrame::Panel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Sunken</enum>
              </property>
              <property name="lineWidth">
               <number>1</number>
              </property>
              <property name="midLineWidth">
               <number>0</number>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_5">
               <item>
                <widget class="QLabel" name="label_8">
                 <property name="maximumSize">
                  <size>
                   <width>56</width>
                   <height>56</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">border-style:none;</string>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                 <property name="pixmap">
                  <pixmap>images/icons8-rooms-64.png</pixmap>
                 </property>
                 <property name="scaledContents">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="bedTypeLabel">
                 <property name="styleSheet">
                  <string notr="true">border-width: 1;
border-style: none;
font-size: 20px;
border-bottom: 2px solid rgba(105,118,132,255);
border-radius: 10;
background-color: white;
</string>
                 </property>
                 <property name="text">
                  <string>1 twins bed</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QFrame" name="checkinFrame_7">
              <property name="styleSheet">
               <string notr="true">border-width: 1;
border-style: none;
border-bottom: 2px solid rgba(105,118,132,255);
border-left: 2px solid rgba(105,118,132,255);
border-radius: 10;
background-color: white;
</string>
              </property>
              <property name="frameShape">
               <enum>QFrame::Panel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Sunken</enum>
              </property>
              <property name="lineWidth">
               <number>1</number>
              </property>
              <property name="midLineWidth">
               <number>0</number>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_7">
               <item>
                <widget class="QLabel" name="label_9">
                 <property name="maximumSize">
                  <size>
                   <width>56</width>
                   <height>56</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">border: none;</string>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                 <property name="pixmap">
                  <pixmap>images/bill.png</pixmap>
                 </property>
                 <property name="scaledContents">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="billLabel">
                 <property name="font">
                  <font>
                   <pointsize>15</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string>$800</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item alignment="Qt::AlignHCenter">
       <widget class="QPushButton" name="btn">
        <property name="font">
         <font>
          <pointsize>15</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="acceptDrops">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton
{
  	border-style: none;
	color: white;
	font-weight: bold;
	background-color: #3f4773;
	border-radius: 25;
	text-align: center;
	padding: 15 25;
}
QPushButton:hover
{
  	background-color:#485187;
}</string>
        </property>
        <property name="text">
         <string>CHECK IN</string>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
